src/ScoreBuffer.h

```diff
@@ -102,9 +104,9 @@ class ScoreBuffer {
 
 		m_DirBuffer = new char[swBatchSize];
 
-		m_EnableBS = false;
+		//m_EnableBS = false;
```

Comment:
Not sure if commenting this is required in the header. It currently does work with the comments, though.